docs/assets/images/expensify-help.svg

```diff
@@ -1 +1 @@
.8,2.6,2.6 h-5.2C54,11.6,55,11,56.2,11L56.2,11z" class="st0"/><path d="M73.9,7.3c-1.7,0-3.2,0.8-4.2,2.2V7.7h-4.7v15.4h5v-8.6c0-2.1,1.1-3,2.4-3c1.5,0,2.3,0.6,2.3,3v8.6h5v-9 C79.7,9.2,77.7,7.3,73.9,7.3L73.9,7.3z" class="st0"/><path d="M97.9,0.7c-1.5,0-2.6,1.1-2.6,2.6c0,1.4,1.1,2.6,2.6,2.6c1.5,0,2.6-1.2,2.6-2.6C100.5,1.8,99.5,0.7,97.9,0.7z" class="st0"/><path d="M100.4,7.7h-5v15.4h5V7.7z" class="st0"/><path d="M88.9,13.5l-1.2-0.3c-1.3-0.3-1.8-0.9-1.8-1.4c0-0.8,0.7-1.4,1.9-1.4c1.5,0,2.2,0.5,2.2,1.9h4.3 c-0.1-3.2-2-5.1-6.8-5.1c-4.1,0-6.6,1.5-6.6,5c0,2.7,1.8,3.9,6,4.9l1.2,0.3c1.3,0.3,1.8,0.9,1.8,1.5c0,0.8-0.8,1.2-2.1,1.2 c-1.7,0-2.5-0.6-2.7-2h-4.9c0.3,3.7,2.9,5.6,7.4,5.6c4.5,0,7.2-1.7,7.2-5.1C94.9,15.7,93.1,14.4,88.9,13.5L88.9,13.5z" class="st0"/><path d="M119.9,7.7L119.9,7.7l-2.8,9.4l-2.7-9.4h-6.5V7.2c0-1.3,0.9-2.5,2.4-2.5h0.7V0.6h-1c-4.3,0.1-7.1,3.1-7.1,6.9 v0.2h-1.9v4.5h1.9v10.8h5V12.3h2.8l3.7,10.4l-0.6,1.6c-0.5,1.2-1,1.5-2,1.5c-0.5,0-1.1,0-1.8-0.2l-0.3,4.1 c0.7,0.2,1.7,0.3,2.9,0.3c3,0,4.5-1.3,6-5.2L125,7.7L119.9,7.7L119.9,7.7z" class="st0"/></g><g><path d="M144.5,23.1c2.7,0,4.3-0.3,4.3-1.1c0-0.7-0.7-0.9-2.1-0.9c-0.4-0.1-0.5-1.5-0.5-3.2V6.9 c0-1.7,0.1-3.1,0.5-3.2c1.5-0.1,2.1-0.3,2.1-0.9c0-0.9-1.6-1.2-4.3-1.2c-2.8,0-4.4,0.3-4.4,1.2c0,0.6,0.7,0.8,2.1,0.9 c0.4,0.1,0.5,1.6,0.5,3.2v3.9c-0.1,0.3-0.3,0.4-1.1,0.4h-8.5c-0.8,0-1-0.1-1.1-0.4V6.9c0-1.7,0.1-3.1,0.5-3.2 c1.5-0.1,2.1-0.3,2.1-0.9c0-0.9-1.6-1.2-4.4-1.2c-2.7,0-4.3,0.3-4.3,1.2c0,0.6,0.7,0.8,2.1,0.9c0.4,0.1,0.4,1.6,0.4,3.2v10.9 c0,1.7-0.1,3.1-0.4,3.2c-1.5,0.1-2.1,0.3-2.1,0.9c0,0.8,1.6,1.1,4.3,1.1c2.8,0,4.4-0.3,4.4-1.1c0-0.7-0.7-0.9-2.1-0.9 c-0.4-0.1-0.5-1.5-0.5-3.2v-4.3c0.1-0.3,0.3-0.4,1.1-0.4h8.5c0.8,0,0.9,0.1,1.1,0.4v4.2c0,1.7-0.1,3.1-0.5,3.2 c-1.5,0.1-2.1,0.3-2.1,0.9C140.1,22.8,141.7,23.1,144.5,23.1z" class="st1"/><path d="M156.3,23.4c2.7,0,4.6-1.2,5.7-2.5c0.7-0.7,1-1.5,1-2c0-0.5-0.2-0.8-0.6-0.8c-0.3,0-0.5,0.2-0.8,0.5 c-0.9,0.9-2.2,1.9-4.3,1.9c-3,0-4.8-2.1-5-5.3h8.8c1.2,0,1.8-0.9,1.8-2.2c0-2.6-2.3-5.2-6.1-5.2c-4.1,0-7.5,3.3-7.5,8.1 C149.3,20.3,152.1,23.4,156.3,23.4z M152.4,13.6c0.4-2.6,2-4.2,4.3-4.2c2.1,0,3.4,1.4,3.4,3c0,0.8-0.5,1.2-1.5,1.2L152.4,13.6z" class="st1"/><path d="M167.6,23.1c2.3,0,3.7-0.3,3.7-1.1c0-0.6-0.6-0.7-1.7-0.8c-0.3-0.1-0.4-1.2-0.4-3.1V7.5 c0-2.1,0.1-3.8,0.1-4.8c0-1.1-0.2-1.6-1-1.6c-0.8,0-3,0.5-4.1,1c-0.4,0.2-0.6,0.4-0.6,0.8c0,0.4,0.4,0.7,2,0.8 c0.4,0.2,0.4,1.9,0.4,3.3V18c0,2.1,0,3-0.4,3.1c-1.1,0.1-1.7,0.2-1.7,0.8C163.9,22.8,165.2,23.1,167.6,23.1z" class="st1"/><path d="M175.7,28.6c2.9,0,4.4-0.2,4.4-1.1c0-0.6-0.6-0.7-2.3-0.9c-0.4-0.2-0.5-1.2-0.5-3.1v-1.9c1,1,2.6,1.7,4.6,1.7 c4.2,0,7.1-3.4,7.1-8.1c0-4.6-2.6-7.5-6.1-7.5c-2.5,0-4.7,1.5-5.6,3.6c0-0.8,0-1.4,0-1.8c0-1.1-0.2-1.6-1-1.6c-0.8,0-3,0.5-4,1 c-0.4,0.2-0.6,0.5-0.6,0.8c0,0.4,0.4,0.7,1.9,0.8c0.4,0.2,0.4,2,0.4,3.2v9.7c0,2.1-0.1,3-0.4,3.1c-1.2,0.1-1.8,0.3-1.8,0.9 C171.9,28.3,173.3,28.6,175.7,28.6z M177.3,16.2c0-4.4,2.2-6.2,4.3-6.2c2.3,0,4,2.2,4,5.9c0,3.6-1.6,5.8-4.2,5.8 C179.1,21.7,177.3,20.2,177.3,16.2z" class="st1"/></g></g></svg>
```

Comment:
```css
Added a media query here.

<style type="text/css">
    .st0{
         fill:#002e22
    }
    @media(prefers-color-scheme: dark){
        .st0{
            fill:#fffaf0
        }
    }
    .st1{
        fill:#03d47c
    }
</style>
```

Reply:
Ah, nice. Didn't know you can do that! 

Can we change the fill color for the wordmark in dark mode to be `E7ECE9` please? Thanks!